keps/sig-release/3344-refine-release-team/README.md

```diff
@@ -0,0 +1,894 @@
+This roster is used to assemble release teams instead of a new shadow survey each time.
+
+<!--
+This section is for explicitly listing the motivation, goals, and non-goals of
+this KEP.  Describe why the change is important and the benefits to users. The
+motivation section can optionally provide links to [experience reports] to
+demonstrate the interest in a KEP within the wider Kubernetes community.
+
+[experience reports]: https://github.com/golang/go/wiki/ExperienceReports
+-->
+
+### Goals
+<!--
+List the specific goals of the KEP. What is it trying to achieve? How will we
+know that this has succeeded?
+-->
+
+- Continue to provide an "on ramp" for new community members.
+- Provide a more streamlined process for release team shadows, including easier return into future release teams.
+- Provide a "stable" position that a member may continue to be on release teams if they wish.
+- Better handle contributors who wish to take breaks from the team, and then return. 
+
+### Non-Goals
+
+<!--
+What is out of scope for this KEP? Listing non-goals helps to focus discussion
+and make progress.
+-->
+
+- Substantively change the makeup and roles of the release team itself.
+- Change how the release team lead, emeritus adviser, and role leads are selected.
+
+## Proposal
+
+<!--
+This is where we get down to the specifics of what the proposal actually is.
+This should have enough detail that reviewers can understand exactly what
+you're proposing, but should not include things like API designs or
+implementation. What is the desired outcome and how do we measure success?.
+The "Design Details" section below is for the real
+nitty-gritty.
+-->
+
+### Release Team Roster
+
+We will add to SIG Release a "Release Team Roster".
+The list of roster members will be maintained in a YAML file in https://github.com/kubernetes/sig-release under a new `release-team/roster` directory.
+People on the roster can be in one of three roles: Shadow, Member, or Emeritus.
+
+At the time of creation, all former role leads or release leads from any release will be added as emeritus members, or a member if they want.
+The 1.25 EA will lead discussions around which shadows from 1.25 should be placed at the member or shadow level on the release team roster.
+
+Roster Shadows may leave at any time by opening a PR to remove themselves.
+Shadows that do not join a release team for two consecutive releases will generally be removed (although this is at the discretion of SIG Release).
+Members may leave at any time by opening a PR to move themselves to the Emeritus section.
+
+### Release Team Changes
+
+The "Lead Shadow" role will be renamed to "Assistant Lead".
+There is otherwise no change in the nature of this role.
+This is to avoid confusion as to what a "shadow" is on the release team, as a "lead shadow" is significantly different to a role shadow.
+
+### Shadow Survey
+
+The per-release shadow selection survey will be replaced with a rolling "expression of interest" form.
```

Comment:
I think we should do both: Selecting new shadows and re-evaluating the existing roster. This makes it easier for new contributors to join because the folks from the roster will not use the shadow form any more.